recipes/tqdist/1.0.0/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: tqdist
+  version: "1.0.0"
+
+source:
+  fn: tqDist-1.0.0.tar.gz
+  url: http://users-cs.au.dk/cstorm/software/tqdist/files/tqDist-1.0.0.tar.gz
+  md5: dc95f3d1672eb1be9f644eb66c41ad86
+  patches:
+   - patch.txt
+   
+requirements:
+  build:
+    - cmake
+    - gcc
```

Comment:
Can you adopt this to fit our guidelines here: https://bioconda.github.io/guidelines.html#c-c
```
    - gcc   # [not osx]
    - llvm  # [osx]
```